lib/gltf.g.dart

```diff
@@ -0,0 +1,4 @@
+// AUTOGENERATED - DO NOT EDIT BY HAND
```

Comment:
I'm not sure this file needs to be checked into source control.  Maybe remove this and add it to `.gitignore`?  I know that's different from how we treat `ISSUES.md`, but that's a user-facing file that we want processed as markdown by GitHub, whereas this can be expected to be built as needed.

Reply:
It's better to check it into git for now. Otherwise, installation process would require more user actions. When `pub get` will be able to trigger build scripts automatically, we can revisit this.

Reply:
Sounds good.  Would you mind adding a line in `.gitattributes` that has `lib/gltf.g.dart eol=lf`? It's helpful for Windows users.  Thanks.